app/helpers/days_helper.rb

```diff
@@ -19,4 +19,13 @@ def day_status_css(d)
   def feedback_submitted?(day)
     student? && current_user.day_feedbacks.find_by(day: day.to_s)
   end
+
+  def total_cohort_students
+    @total ||= cohort.students.length
```

Comment:
I think this instance variable should have a more descriptive name than `@total`. It is shared by all the methods in this class.